### ISSUE TITLE
Trying to copy text with left click + drag, it will sometimes grab an old image instead

### ISSUE THREAD

**Issue by inconspiciousfbivan** [ccd0/4chan-x#2031]
On trying to copy text, sometimes it will end up grabbing an image, sometimes an image that was on the front page (while I am on a new thread) and I am forced to either drop the image which will force the page to display the image, or I can try to right-click at the same time to open options then click off and try again with the left click text copy. I frequently have 6-10 pages open.

**Comment by inconspiciousfbivan** [comment]
> 
> 
> there's Copy text in the Menu

True, but that doesn't fix the problem, nor does it fix when dragging an image will grab an old image and lock the cursor to said old image.

**Comment by ubone** [comment]
what os/browser/script-addon

**Comment by inconspiciousfbivan** [comment]
Firefox, latest version. WIN 10 using Greasemonkey, noscript, APB, racism
simulator
